packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -295,10 +295,11 @@ export class MessageWidget extends Component {
     const {props} = this;
     const handleEvent = this.handleEvent;
     const toUser = space.toPerson;
+    const currentUserId = props.user.get(`currentUser`).id;
```

Comment:
I like the pattern you used for the webrtc status check. We may want to move this stuff into the selector as well since we use it everywhere. Just have to be careful and make sure we’re memoizing correctly

Reply:
Yea, I think that should be part of the render optimizations. IMO, we shouldn't be calling any immutable `get` methods in render.